project_euler/problem_117/sol1.py

```diff
@@ -0,0 +1,27 @@
+"""
+Project Euler Problem 104: https://projecteuler.net/problem=117
+Using a combination of grey square tiles and oblong tiles chosen from:
+red tiles (measuring two units), green tiles (measuring three units),
+and blue tiles (measuring four units), it is possible to tile a row
+measuring five units in length in exactly fifteen different ways.
+
+How many ways can a row n units long be filled with
+"""
+
+
+def solution(length: int = 50) -> str:
+    """
+    >>> solution(50)
+    '100808458960497'
+    """
```

Comment:
Please read through the [coding style](https://github.com/TheAlgorithms/Python/blob/master/project_euler/README.md#coding-style) point 5 about `doctest`.

Reply:
so I just need to add a brief explanation about what does function can do.. AmI right?